lib/ansible/cli/galaxy.py

```diff
@@ -385,6 +385,8 @@ def add_verify_options(self, parser, parents=None):
                                         'canonical manifest hash.')
         verify_parser.add_argument('-r', '--requirements-file', dest='requirements',
                                    help='A file containing a list of collections to be verified.')
+        verify_parser.add_argument('--keyring', dest='keyring', default='~/.ansible/pubring.kbx',
```

Comment:
Does `~` need to be resolved at this stage?

Reply:
Good catch. Now this is getting resolved before passing it to the ConcreteArtifactsManager instance so it's the same whether the default is used or if it's provided.